src/layouts/components/Editor/TableBubbleMenu.tsx

```diff
@@ -0,0 +1,63 @@
+import { Box } from "@chakra-ui/react"
+import { BubbleMenu } from "@tiptap/react"
+import remixiconUrl from "remixicon/fonts/remixicon.symbol.svg"
+
+import { useEditorContext } from "contexts/EditorContext"
+
+export const TableBubbleMenu = () => {
+  const { editor } = useEditorContext()
+
+  return (
+    <BubbleMenu
+      shouldShow={() => editor.isActive("table") && !editor.isActive("link")}
```

Comment:
Should have one central utility function to determine this. Otherwise if we add new bubble menus in the future we got n places to update.

Reply:
this is intended - different bubble menus shuold show based on their surrounding context. unless you mean something like `showBubbleMenu("table")` and `showBubbleMenu("link")`?

Reply:
Yup, so we only have a central location (i.e. `showBubbleMenu`) to edit when we add more bubble menus in the future.